index.json

```diff
@@ -108,6 +108,13 @@
       "license": "GPL-3.0-or-later",
       "type": "Modpack",
       "url": "mods/SG4.json"
+    },
+    {
+      "name": "Chaos52Dingo - 9th May 2025",
+      "version": "2025-05-09-8a67d16",
+      "license": "GPL-3.0-or-later",
+      "type": "Modpack",
+      "url": "mods/Chaos-2025-05-09-8a67d16.json"
```

Comment:
What's the reason for copying the modpack instead of pointing directly to the json in the branch? (like Sextant and Aviation)

Reply:
I didn't know that was an option, I just used LTX as a template since that's what the ruleset was based on.

Reply:
Let's avoid duplication then

Reply:
Does it matter that it's a `.modpack` rather than a `,json`?

Reply:
Hm, the fc21 fcmp doesn't support the `.modpack` format anymore... maybe we should've tested it first? :smile:

Reply:
No testing, only pushing!